src/test/java/uk/nhs/hee/tis/trainee/sync/service/PlacementSpecialtySyncServiceTest.java

```diff
@@ -69,6 +73,8 @@ class PlacementSpecialtySyncServiceTest {
   private static final String PLACEMENT_SPECIALTY_PLACEMENT_ID = "placementId";
 
   private static final String PLACEMENT_SPECIALTY_DATA_SPECIALTY_TYPE_PRIMARY = "PRIMARY";
+  private static final String PLACEMENT_SPECIALTY_DATA_SPECIALTY_TYPE_SUB_SPECIALTY =
+      "SUB_SPECIALTY";
   private static final String PLACEMENT_SPECIALTY_DATA_SPECIALTY_TYPE_NOT_PRIMARY = "";
```

Comment:
I'd think of renaming this to `PLACEMENT_SPECIALTY_DATA_SPECIALTY_TYPE_OTHER` - that is the only other type at the moment, apart from primary and sub-specialty.